### ISSUE TITLE
qubes-core-agent-thunar is missing files in the Debian 11 release

### ISSUE THREAD

**Issue by ghost** [QubesOS/qubes-issues#7102]
### Qubes OS release

4.0

### Brief summary

Debian 11 package `qubes-core-agent-thunar` does not provide all the files, so the functionality to copy to other vms is not available in Thunar.

### Steps to reproduce

* Use the Debian 11 Qubes os template
*  Install `qubes-core-agent-thunar` and `thunar`
*  Start `thunar`
*  Check if the copy vm functionality is there
* (run `dpkg-query -L qubes-core-agent-thunar` to verify the files)

### Expected behavior

* Have copy vm functionality in thunar
* (See files like `uca_qubes.xml` when listing the package files)

### Actual behavior

* No copying to other app vms possible
* Output of  `dpkg-query -L qubes-core-agent-thunar`
```
$ `dpkg-query -L qubes-core-agent-thunar
/etc
/etc/xdg
/etc/xdg/xfce4
/etc/xdg/xfce4/xfconf
/etc/xdg/xfce4/xfconf/xfce-per-channel-xml
/etc/xdg/xfce4/xfconf/xfce-per-channel-xml/thunar.xml
```

**Comment by ekaflaer** [comment]
I have upgraded to Qubes OS 4.1 and the package has all the necessary files there and the functionality in Thunar is working as expected.

**Comment by unman** [comment]
This issue is resolved in 4.1, and can be closed.
@andrewdavidwong

**Comment by fepitre** [comment]
Closing as resolved. If anyone believes this issue is not yet resolved, or if anyone is still affected by this issue, please leave a comment, and we'll be happy to reopen it. Thank you.
